parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -269,6 +269,14 @@ write_files:
     [Service]
     ExecStartPost=/sbin/iptables -P FORWARD ACCEPT
     #EOF
+
+- path: /etc/crictl.yaml
```

Comment:
make this optional for containerd only?

Reply:
this is already in the {{if NeedsContainerd}} block